web/src/stream_popover.ts

```diff
@@ -516,23 +516,55 @@ export async function build_move_topic_to_stream_popover(
     }
 
     let curr_selected_stream: number;
-    // Warn if any of current topic participants are NOT subscribed
+
+    function get_messages_to_be_moved(
+        propagate_mode: string,
+        selected_message: Message | undefined,
+    ): Message[] {
+        const all_locally_cached_conversation_messages = message_util.get_loaded_messages_in_topic(
+            current_stream_id,
+            topic_name,
+        );
+
+        // It's move-topic modal, so no message is selected.
+        if (selected_message === undefined) {
+            return all_locally_cached_conversation_messages;
+        }
```

Comment:
Logically, this check is not needed. But I think it's better to be clear/explicit, meaning we specifically return all messages because it's a move-topic  modal.